llvm/test/Transforms/LoopVectorize/AArch64/interleaved_store.ll

```diff
@@ -0,0 +1,117 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 6
+; RUN: opt -passes=loop-vectorize -enable-interleaved-mem-accesses=true -max-interleave-group-factor=16  -S < %s | FileCheck %s
```

Comment:
`-enable-interleaved-mem-accesses=true` should not be needed, why does the max factor need to capped?